board/templatetags/get_page_url.py

```diff
@@ -0,0 +1,19 @@
+from django import template
+
+
+register = template.Library()
+
+
+@register.filter
+def get_page_url(page_number, filter_initials):
+    platforms = filter_initials['platforms']
+    genres = filter_initials['genres']
+    rating = filter_initials['rating']
+    url = '?'
+    for platform in platforms:
+        url += f'platforms={platform}' + '&'
+    for genre in genres:
+        url += f'genres={genre}' + '&'
+    url += f'rating={rating}' + '&'
+    url += f'page={page_number}'
+    return url
```

Comment:
What if we would like to add one more filter parameter? With current approach this would affect this tag logic as well. Check this solution instead: https://stackoverflow.com/questions/2047622/how-to-paginate-django-with-other-get-variables